espanol/pelea_epica/pelea_epica.md

```diff
@@ -0,0 +1 @@
+Puso su mano derecha en su frente y se teletransportó al medio del ejército enemigo. La épica batalla estaba apunto de empezar.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.